test/minikube/minikube_long_journal_migration_test.go

```diff
@@ -0,0 +1,123 @@
+// +build diagnostics
+// not the ideal plan, but diagnostics is currently 1/2 the time of other plans
+
+package minikube
+
+import (
+	"fmt"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/nuodb/nuodb-helm-charts/v3/test/testlib"
+	"github.com/stretchr/testify/require"
+	"os"
+	"testing"
+)
+
+func TestChangingJournalLocationFails(t *testing.T) {
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+
+	options := helm.Options{}
+
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, &options, 1, "")
+
+	admin0 := fmt.Sprintf("%s-nuodb-cluster0-0", helmChartReleaseName)
+
+	t.Run("startDatabaseStatefulSet", func(t *testing.T) {
+		defer testlib.Teardown(testlib.TEARDOWN_DATABASE)
+
+		options := helm.Options{
+			SetValues: map[string]string{
+				"database.sm.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+				"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+				"database.te.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+				"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+				"database.sm.hotCopy.journalPath.enabled": "false",
+			},
+		}
+
+		databaseReleaseName := testlib.StartDatabase(t, namespaceName, admin0, &options)
+
+		options.SetValues["database.sm.hotCopy.journalPath.enabled"] = "true"
+
+		err := helm.UpgradeE(t, &options, testlib.DATABASE_HELM_CHART_PATH, databaseReleaseName)
+		require.Error(t, err)
```

Comment:
What the error will look like? I suppose that Helm will complain that the PVC are immutable. Is there a reason to assert the message?

Reply:
yeah, I did not check for the exact message.